server/core/src/main/java/org/infinispan/server/core/LifecycleCallbacks.java

```diff
@@ -60,6 +67,23 @@ public void cacheManagerStarting(GlobalComponentRegistry gcr, GlobalConfiguratio
          BinaryTranscoder transcoder = encoderRegistry.getTranscoder(BinaryTranscoder.class);
          transcoder.overrideMarshaller(jbossMarshaller);
       }
+
+      ComponentRef<ExecutorService> executorServiceRef = basicComponentRegistry.getComponent(
+            KnownComponentNames.NON_BLOCKING_EXECUTOR, ExecutorService.class);
+      ExecutorService oldNonBlockingExecutor = executorServiceRef.wired();
+
+      String nodeName = globalConfiguration.transport().nodeName();
+
+      EventLoopGroup eventLoopGroup = NettyTransport.buildEventLoop(Runtime.getRuntime().availableProcessors(),
```

Comment:
should  `config.ioThread()` or at least, its default value `2 * ProcessorInfo.availableProcessors()`

Reply:
I am trying to move away from that config as much as possible. The only reason that if statement is there is because of some test code that is a PIT butt to refactor out. But I forgot to use ProcessorInfo.availableProcessors. I can use that.